### DISCUSSION TITLE
Which element do I have to use for pagination?

### DISCUSSION THREAD

**Discussion by howpagination** [AutomaApp/automa#1204] [General]
Hi.

I have a pagination that looks like below:

[ 1 ] [ 2 ] [ 3 ] [ 4 ] [ 5 ] [ 6 ] [ 7 ] [ 8 ] [ 9 ] [ 10 ]  NEXT >

Which element do I have to use for clicking each page number?

Also, please give me some instructions how to use the element for it.

Thank you very much. : )

**Comment by flashvnn** [comment]
You should use Javascript block and add script that get all pagination urls (try google with jQuery) then storage to variable or table. Then use loop variables to open each url and get your data
